databricks/sdk/runtime/__init__.py

```diff
@@ -16,6 +16,18 @@
 except ImportError:
```

Comment:
We should not append init_runtime_native_auth to dbruntime_objects

Reply:
this is the part that @xiaochen-db is driving, outside of scope of this PR :)